CMakeLists.txt

```diff
@@ -2,33 +2,45 @@ cmake_minimum_required (VERSION 3.11)
 
 add_subdirectory(third_party)
 
-project(smk-starter
+project(lysy-bingo
```

Comment:
does this work for you? I had some problems earlier that it generated lysy-bingo.js but tried to include smk-starter.js in html

Reply:
Have you tried changing the linked .js file in index.html?
Change
`<script async src="smk-starter.js"></script>`
To
`<script async src="lysy-bingo.js"></script>`

Reply:
Yeah I did that, but I had to change it every time I ran `emcmake cmake ..` on an empty build dir